core/email.py

```diff
@@ -175,6 +174,49 @@ def email_from_admin(username, subject, message, html=False):
                       cc=[email_address_str(from_name, from_email)],
                       html=html)
 
+def send_approved_quota_email(user, request, reason):
+    """
+    Notify the user the that their request has been approved.
+    """
+    template = """
```

Comment:
Is there a way to write the email text in `"""text"""` while still maintaining visual indentation?  This looks hideous.